addons/udes_stock/models/stock_location.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import os.path
+from odoo import fields, models
+
+
+class StockLocation(models.Model):
+    _name = "stock.location"
+    # Add messages and abstract model to locations
+    _inherit = ["stock.location", "mail.thread", "mixin.stock.model"]
+
+    # Disable translation instead of renaming.
+    name = fields.Char(translate=False)
```

Comment:
I think there is a way of overriding a method from ir_model_fields i think that can add/modify attribute for existing fields instead of redefining the field again. In that way can have smth more generic helper name of the field, model, attribute , value example and when overriding the method to pick all changes from helpers.
Just imagining now :)

Solving the comment though...